contracts/seed/Seed.sol

```diff
@@ -24,26 +24,26 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
  */
 contract Seed {
     using SafeMath for uint256;
-    using SafeMath for uint16;
+    using SafeMath for uint32;
     using SafeMath for uint8;
 
     // Locked parameters
     address public beneficiary;
     address public admin;
     uint256 public softCap;
     uint256 public hardCap;
-    uint256 public seedRemainder;
+    uint256 public seedAmountAtStart;   // Amount of seed at the start of distribution
```

Comment:
including fees, or not?  The original intention was not

Reply:
Have added this to a new Issue. #160. This change needs some test changes too, so it's better to move it to a new branch because this PR is very big now.